src/GRANULAR/gran_sub_mod_tangential.cpp

```diff
@@ -123,16 +123,29 @@ void GranSubModTangentialLinearHistory::calculate_forces()
 
   // rotate and update displacements / force.
   // see e.g. eq. 17 of Luding, Gran. Matter 2008, v10,p235
+
+  // int newapproach = 0; //DRV: Original Method results in unphysical pendular motion
+  int newapproach = 1; //DRV: New Method
+
   if (gm->history_update) {
-    rsht = dot3(history, gm->nx);
+    if(newapproach == 0){
```

Comment:
The LAMMPS programming style has a blank between "if" and "(", also ")" and "{"

Reply:
Thanks, have incorporated these changes.